ci/build_container/build_recipes/cares.sh

```diff
@@ -10,10 +10,26 @@ VERSION=cares-1_14_0
 CPPFLAGS="$(for f in $CXXFLAGS; do if [[ $f =~ -D.* ]]; then echo $f; fi; done | tr '\n' ' ')"
 CFLAGS="$(for f in $CXXFLAGS; do if [[ ! $f =~ -D.* ]]; then echo $f; fi; done | tr '\n' ' ')"
 
-wget -O c-ares-"$VERSION".tar.gz https://github.com/c-ares/c-ares/archive/"$VERSION".tar.gz
+curl https://github.com/c-ares/c-ares/archive/"$VERSION".tar.gz -sLo c-ares-"$VERSION".tar.gz
 tar xf c-ares-"$VERSION".tar.gz
 cd c-ares-"$VERSION"
-./buildconf
-./configure --prefix="$THIRDPARTY_BUILD" --enable-shared=no --enable-lib-only \
-  --enable-debug --enable-optimize
-make V=1 install
+
+mkdir build
+cd build
+
+build_type=RelWithDebInfo
```

Comment:
I'm wondering if there have been any noticeable changes in performance or build object when switching from the `configure` flags like `--enable-optimize` to the `build_type` here.

Reply:
Using `configure`, it looks like the compiler flags were:

```
<c>   DW_AT_producer    : (indirect string, offset: 0x6747): GNU C11 5.4.0 20160609 -mtune=generic -march=x86-64 -ggdb3 -g -O2 -fvisibility=
hidden -fno-omit-frame-pointer -fPIC -fstack-protector-strong
```

using `ninja`, it looks like the compiler flags were:

```
<c>   DW_AT_producer    : (indirect string, offset: 0x602f): GNU C11 5.4.0 20160609 -mtune=generic -march=x86-64 -ggdb3 -g -O2 -O2 -fno-omit-frame-pointer -fstack-protector-strong
```

We got these compiler flags by running `objdump --dwarf libcares.a | grep DW_AT_producer`

What is the usual way to catch a performance regression?

Reply:
We don't have a standardized way to do this. I would flag this in the "Risk level" as medium and point out that there is potential for performance regression. I think it should be fine based on those flags (?).

Reply:
Done